internal/graph/check.go

```diff
@@ -218,7 +218,6 @@ func resolver(ctx context.Context, concurrencyLimit uint32, resultChan chan<- ch
 // union implements a CheckFuncReducer that requires any of the provided CheckHandlerFunc to resolve
 // to an allowed outcome. The first allowed outcome causes premature termination of the reducer.
 func union(ctx context.Context, concurrencyLimit uint32, handlers ...CheckHandlerFunc) (*ResolveCheckResponse, error) {
-
```

Comment:
`intersection` and `exclusion` have a leading newline. Please either make consistent or add the visual whitespace back.